runner/wasm/wasmtime/x.go

```diff
@@ -27,16 +27,6 @@ func TransInBytesParam(store *wasmtime.Store, instance *wasmtime.Instance, in []
 	return aret, size, size
 }
 
-func TransOutBytesReturn(store *wasmtime.Store, instance *wasmtime.Instance, cap int32) (_, _, _ interface{}) {
```

Comment:
this was dead code and also interrupt changed too much so it needs to be redone

https://github.com/bytecodealliance/wasmtime-go/pull/124

Reply:
I got the upgrade notify recently by `renovate` bot.
Should we keep with old wasmtime version, and waitting for another version?
I am  worring about wasmtime api stability.

@codefromthecrypt

Reply:
wasmtime release policy is monthly so I think it best to just accept whatever happens.